packages/theme-chalk/src/mixins/_var.scss

```diff
@@ -0,0 +1,13 @@
+@mixin set-css-color-primary($i) {
```

Comment:
I didn't see the `@use "sass:map"` declaration here, is that ok for using this without explicitly importing?

Reply:
Yeah, it can work. Because it is only called in `common/var.scss` which has `sass:map`.

Of course, just in case, I can add it later.

Reply:
So for better consistency here I think we need to explicitly add it so that would not cause any misleading.